src/cluster/src/install.rs

```diff
@@ -597,17 +597,28 @@ impl ClusterInstaller {
     ///  1. Check if compatible helm version is installed
     ///  2. Check if compatible sys charts are installed
     ///  3. Check if the K8 config hostname is not an IP address
-    fn pre_install_check(&self) -> Result<(), CheckError> {
+    async fn pre_install_check(&self) -> Result<(), CheckError> {
         check_helm_version(&self.helm_client, DEFAULT_HELM_VERSION)?;
         check_system_chart(&self.helm_client, DEFAULT_CHART_SYS_REPO)?;
         check_already_installed(&self.helm_client, DEFAULT_CHART_APP_REPO)?;
         check_cluster_server_host()?;
+        check_load_balancer_status().await?;
+        Ok(())
+    }
+
+    async fn _try_minikube_tunnel(&self) -> Result<(), ClusterError> {
+        // run minikube tunnel  > /tmp/tunnel.out 2> /tmp/tunnel.out
+        Command::new("minikube")
+            .arg("tunnel")
+            .args(&[">", "/tmp/tunnel.out", "2>", "/tmp/tunnel.out"])
```

Comment:
Does this actually work? The `>` syntax is a bash feature, not an argument to minikube. Is there any reason to keep the tunnel output around or can we just discard it? If we can discard it, that will happen by default if we just don't inherit the output of this command.
@sehz what should be the desired output behavior for minikube tunnel? Discard or pipe to tunnel file?

Reply:
pipe to file so it can be debugged later

Reply:
This is incorrect.   It should be using `Stdio` redirect instead of doing in the args